### PR TITLE
Pp 9236 run pact provider tests

### PR DESCRIPTION
**Pull request by oswaldquek**
We can see the "Run Ledger as Provider" and "Run Connector as Provider" checks on this PR run successfully.

### CODE REVIEW

File: .github/workflows/post-merge.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Post Merge
 
 on:
+  pull_request:
```
Comment (oswaldquek) [inline .github/workflows/post-merge.yml:4]: will be reverted when approved

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Github Actions Tests
 
 on:
   workflow_call:
-  pull_request:
+#  pull_request:
```
Comment (oswaldquek) [inline .github/workflows/run-tests.yml:5]: will be reverted when approved

**Review by jfharden — COMMENTED** [review]
Looks good, if you tidy I'll approve 👍

**Comment by oswaldquek** [comment]
> Looks good, if you tidy I'll approve 👍

Done. Thanks!

**Review by jfharden — APPROVED** [review]

**Review by jfharden — APPROVED** [review]

**Pull request merged**